master/usage/ipv6.md

```diff
@@ -1,31 +1,158 @@
 ---
-title: IPv6 Support
+title: Enabling IPv6 Support
 canonical_url: 'https://docs.projectcalico.org/v3.0/usage/ipv6'
 ---
 
-{{site.prodname}} supports connectivity over IPv6, between compute hosts, and
-between compute hosts and their VMs. This means that, subject to
-security configuration, a VM can initiate an IPv6 connection to another
-VM, or to an IPv6 destination outside the data center; and that a VM can
-terminate an IPv6 connection from outside.
-
-## Requirements for containers
-
-Containers have no specific requirements for utilizing IPv6
-connectivity.
-
-## Requirements for guest VM images
+### About IPv6 Support
+
+You may be interested in this document if you want to enable IPv6
+Support. This would be because:
+- You want workloads to communicate over IPv6.
+- You want workloads to initiate connections to IPv6 services.
+- You want workloads to terminate incomint IPv6 connections.
+
+### Purpose of this page
+
+Provide guidance on how to enable IPv6 addressing for Calico networked
+workloads.
+
+## Enabling IPv6 with Kubernetes
+
+The sections below are not intended to be a detailed setup guide but
+to layout the settings necessary to enable IPv6 operation of Kubernetes
+and Calico.
+
+### Prerequisites
+
+#### Host Prerequisites
+
+- Each host must have an IPv6 address that is reachable from the other
+  hosts.
+  > **Note:** Each host running Calico currently must also have an IPv4
+  > address due to [bug 1204](https://github.com/projectcalico/calico/issues/1204).
+  {: .alert .alert-info}
+- Each host must have the sysctl setting `net.ipv6.conf.all.forwarding`
+  setting it to `1`.  This ensures both Kubernetes service traffic
+  and Calico traffic is forwarded appropriately.
+- Each host must have a default IPv6 route.
+
+#### Kubernetes Components Prerequisites
+
+Kubernetes components must be configured to operate with IPv6.
+Below are the flags for each Kubernetes component that will help ensure
```

Comment:
why does it say "help"? Makes it sound like you're not giving them all the information they would need to enable ipv6...which is the title and purpose of the page....

what you could do is to rephrase lines 42-43 to this "To enable IPv6, set the following flags." 

That pretty much covers everything with a lot less words IMO

Reply:
It is because I am not confident that the flags laid out are the only ones necessary for Kubernetes to use IPv6.  I was trying to give the impression that this is to help them in setting up K8s with IPv6 and not a comprehensive guide.